### ISSUE TITLE
Rename repository to jamulus.io or "website"

### ISSUE THREAD

**Issue by ann0see** [jamulussoftware/jamulussoftware.github.io#838]
The name jamuluswebsite is long to type and redundant in our organization. I think we should think about renaming this repo to something short like jamulus.io or website.

What do you think?

**Comment by jujudusud** [comment]
I do not need this name change as far as I am concerned.
But I have no objection to changing the name to a shorter and clearer one.
jamulus.io' sounds like a good idea.

**Comment by hoffie** [comment]
I agree that jamulus.io would have been the more natural choice. Changing it now might need some coordination and checking though as people would have to update their Git configs, etc.

I thought the website might have been available at jamuluswebsite.github.io now, but it seems like it isn't so there is nothing to break in that place. :)

**Comment by ann0see** [comment]
>  might have been available at jamuluswebsite.github.io n

It was at the beginning, but it no longer is.
